Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM jupyter/minimal-notebook
+
+MAINTAINER rscalfani@mitre.org
+
+WORKDIR /usr/src/app
+
+RUN pip install pandas
+RUN pip install matplotlib
+RUN pip install ipywidgets
+RUN pip install seaborn
+RUN pip install qgrid
+
+COPY . ./
+
+RUN mkdir growthviz-data/output/
```

Comment:
I think this needs to be a `mkdir -p`. Trying to do a build on my machine is failing.